spec/support/simple_work.rb

```diff
@@ -10,7 +10,9 @@ module Test
     #
     # @example creating with FactoryBot
     #   work = FactoryBot.valkyrie_create(:hyrax_work, :public, title: ['Comet in Moominland'])
-    class SimpleWork < Hyrax::Work; end
+    class SimpleWork < Hyrax::Work
+      include Hyrax::Schema(:basic_metadata)
```

Comment:
do you want to be using `Monograph` instead?

`SimpleWork` is meant to be minimal, ensuring Hyrax works *without* including BasicMetadata (this has been a problem in the past)

Reply:
(it's also possible, i guess, that the problem here is that `WorkShowPresenter` really does only work with BasicMetadata)?

Reply:
@no-reply It's needed for the resource_type field. Maybe that's really a core field?

Reply:
i don't think it should be!

my understanding is that Hyrax promises to work with models that only have the current core properties. if the show presenter will error if it's not present, i'd think that's a bug!

Reply:
okay, you are so very right, monograph works great, thank you for your persistence lol